### ISSUE TITLE
npm start doesn't work on Windows

### ISSUE THREAD

**Issue by AlenPelin** [martpie/monorepo-typescript-next-the-sane-way#2]
Taking into account it is sample repo, don't treat this issue very seriously, but it is worth mentioning in readme it was created on Linux (or MacOS?) and may not work in other environments.

Specifically, executing `npm start` terminates with error:
```
PS C:\art\monorepo-typescript-next-the-sane-way-master\website> npm start

> next-ts-website@1.0.0 start C:\art\monorepo-typescript-next-the-sane-way-master\website
> NODE_ENV=production node .next/production-server/server/index.js

'NODE_ENV' is not recognized as an internal or external command,
operable program or batch file.
```

**Comment by martpie** [comment]
Good point, I can address it easily

**Comment by martpie** [comment]
Fixed in e8ccb339b8adabae4d465ea4aa0692f39df34ab1

Thank you for acknowledging the experimentation character of this repo. My goal is really to come with the best setup possible with the best standard.

In that regard, just fyi, I may change this rule of "one file to rule them all", as each sub-folder should have their own files/config. Even if that means code/config repetition.

Still figuring out what could be the best path :)

**Comment by AlenPelin** [comment]
Thank you, I didn't know about cross-env - nice catch.
